wrapper/src/main/java/software/amazon/jdbc/dialect/AuroraMysqlDialect.java

```diff
@@ -24,7 +24,7 @@
 import java.util.List;
 import software.amazon.jdbc.hostlistprovider.AuroraHostListProvider;
 
-public class AuroraMysqlDialect extends MysqlDialect {
+public class AuroraMysqlDialect extends MysqlDialect implements AuroraDialect {
```

Comment:
It's not necessary to state "implements AuroraDialect" since MysqlDialect already implements it.

Reply:
The AuroraDialect interface was something added in this PR in order to have the Aurora PG and MySQL dialects be able to return the node ID queries.